### PR TITLE
Switch from mount(8) to mount(2) syscall, final

### LINKED ISSUE #90: remove dependency on `mount` binary
https://github.com/rootless-containers/rootlesskit/blob/master/pkg/child/child.go has extra exec for `mount(8)` binary.

`mount(2)` syscall could be called directly.

### PR DESCRIPTION
**Pull request by Zyqsempai**
Fix #90 

Signed-off-by: Boris Popovschi <zyqsempai@mail.ru>

### CODE REVIEW

**Comment by Zyqsempai** [comment]
@AkihiroSuda PTAL

File: pkg/child/hosts.go [hunk 1]
```diff
@@ -56,11 +56,9 @@ func mountEtcHosts(tempDir string) error {
...
 		return errors.Wrapf(err, "writing %s", myEtcHosts)
 	}
-	cmds := [][]string{
-		{"mount", "--bind", myEtcHosts, "/etc/hosts"},
-	}
-	if err := common.Execs(os.Stderr, os.Environ(), cmds); err != nil {
-		return errors.Wrapf(err, "executing %v", cmds)
+
+	if err := unix.Mount(myEtcHosts, "/etc/hosts", "", uintptr(unix.MS_BIND), ""); err != nil {
+		return errors.Wrapf(err, "failed to create bind mount on %s", myEtcHosts)
```
Comment (AkihiroSuda) [inline pkg/child/hosts.go]: nit: "failed to create bind mount /etc/hosts for %s"?

File: pkg/child/resolvconf.go [hunk 2]
```diff
@@ -36,11 +35,9 @@ func mountResolvConf(tempDir, dns string) error {
...
 		return errors.Wrapf(err, "writing %s", myResolvConf)
 	}
-	cmds := [][]string{
-		{"mount", "--bind", myResolvConf, "/etc/resolv.conf"},
-	}
-	if err := common.Execs(os.Stderr, os.Environ(), cmds); err != nil {
-		return errors.Wrapf(err, "executing %v", cmds)
+
+	if err := unix.Mount(myResolvConf, "/etc/resolv.conf", "", uintptr(unix.MS_BIND), ""); err != nil {
+		return errors.Wrapf(err, "failed to create bind mount on %s", myResolvConf)
```
Comment (AkihiroSuda) [inline pkg/child/resolvconf.go]: same as above

**Comment by Zyqsempai** [comment]
@AkihiroSuda Done

**Pull request merged**